apps/web/src/pages/tenants/components/UpdateTenantSidebar.tsx

```diff
@@ -82,61 +98,123 @@ export function UpdateTenantSidebar({
+    onSuccess: async () => {
+      await queryClient.refetchQueries({
+        predicate: ({ queryKey }) => queryKey.includes(QueryKeys.tenantsList),
+      });
+      successMessage(`The tenant ${tenant?.name} is deleted`);
+    },
+    onError: (e: any) => {
+      errorMessage(e.message || 'Unexpected error');
+    },
+  });
+
+  const onDeleteTenant = async (identifier) => {
+    await deleteTenantMutate({
+      identifier: identifier,
+    });
+
+    onClose();
+  };
+
   if (!tenant) {
     return null;
   }
 
   return (
-    <Sidebar
-      isOpened={isOpened}
-      onClose={onClose}
-      isLoading={isLoadingTenant}
-      onSubmit={(e) => {
-        handleSubmit(onUpdateTenant)(e);
-        e.stopPropagation();
-      }}
-      customHeader={
-        <Stack h={80}>
-          <Controller
-            control={control}
-            name="name"
-            defaultValue=""
-            render={({ field }) => {
-              return (
-                <NameInput
-                  {...field}
-                  value={field.value}
-                  data-test-id="tenant-title-name"
-                  placeholder="Enter tenant name"
-                  ml={-10}
-                />
-              );
-            }}
-          />
-        </Stack>
-      }
-      customFooter={
-        <Group position="apart" w="100%">
-          <Stack spacing={0}>
-            <Text size={'sm'} color={colors.B40}>
-              Updated at {format(new Date(tenant.updatedAt), 'dd/MM/yyyy HH:mm')}
-            </Text>
-            <Text size={'sm'} color={colors.B40}>
-              Created at {format(new Date(tenant.createdAt), 'dd/MM/yyyy HH:mm')}
-            </Text>
+    <>
+      <Sidebar
+        isOpened={isOpened}
+        onClose={onClose}
+        isLoading={isLoadingTenant}
+        onSubmit={(e) => {
+          handleSubmit(onUpdateTenant)(e);
+          e.stopPropagation();
+        }}
+        customHeader={
+          <Stack h={80}>
+            <Group position="apart" spacing={120}>
+              <Controller
+                control={control}
+                name="name"
+                defaultValue=""
+                render={({ field }) => {
+                  return (
+                    <NameInput
+                      {...field}
+                      value={field.value}
+                      data-test-id="tenant-title-name"
+                      placeholder="Enter tenant name"
+                      ml={-10}
+                    />
+                  );
+                }}
+              />
+              <Tooltip label={'Delete tenant'} position="bottom">
+                <ActionIcon
+                  onClick={() => {
+                    setModalIsOpened(true);
+                  }}
+                  data-test-id="tenant-delete-btn"
+                  variant="transparent"
+                >
+                  <TrashButton>
+                    <Trash />
+                  </TrashButton>
+                </ActionIcon>
```

Comment:
The icon is not aligned with the x button. 
<img width="422" alt="Screen Shot 2023-10-16 at 12 14 06" src="https://github.com/novuhq/novu/assets/7778680/5a40756c-5975-4fb8-97ab-8b555de1bca0">

Please take a look on how it was done in the update integration sidebar `UpdateIntegrationSidebarHeader.tsx`. This is how it looks, icons are aligned: 
<img width="439" alt="Screen Shot 2023-10-16 at 12 15 38" src="https://github.com/novuhq/novu/assets/7778680/80af2270-6712-4499-b1d7-a2f47d83bac4">